tests/lib/snaps/test-snapd-tuntap/bin/tuntap.py

```diff
@@ -11,7 +11,7 @@
 
 def if_open(dev: str) -> int:
     TUNSETIFF = 0x400454ca
-    TUNSETOWNER = TUNSETIFF + 2
+    # TUNSETOWNER = TUNSETIFF + 2
```

Comment:
unused?

Reply:
It's used in commented out code a little bit below in this function so I left it in for debugging in case we ever need to dig into this again this might be useful to have